### ISSUE TITLE
Train and test inverted

### ISSUE THREAD

**Issue by fcomitani** [scikit-multilearn/scikit-multilearn#129]
Hi,

I may be mistaken, but it seems that the output of `iterative_train_test_split` is inverted.

The expected output of the function is 

```
    Returns
    -------
    X_train, y_train, X_test, y_test
        stratified division into train/test split


```

but if I run it like

`x_train,y_train,x_test,y_test= iterative_train_test_split(df, labels, test_size=0.2)`

I get that x_train has 20% of the data and x_test 80%.

Thanks

**Comment by fcomitani** [comment]
I opened a pull request to revert some changes and fix this. I'm closing this issue.

**Comment by fcomitani** [comment]
This is already solved in the latest version.
